specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -150,28 +150,25 @@
         }
       }
     },
-    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/validateAddress": {
+    "/providers/Microsoft.Billing/validateAddress": {
```

Comment:
Based on the swagger repo, it looks like .../validateAddress is only present in this -preview api. Can you confirm we're not renaming a stable api here.